Assets/Scripts/Input/DragonPlayerController.cs

```diff
@@ -88,4 +107,8 @@ private void HandleDodgeInput(InputAction.CallbackContext context)
             combat.Dodge(direction);
         }
     }
+    protected void HandleDeathEvent()
+    {
+        gameObject.GetComponent<Rigidbody2D>().gravityScale = 1;
```

Comment:
The `ActorController.Movement` component has a reference to the actor's Rigidbody component, so you could do `movement.Rigidbody2d.gravityScale = 1;` for both the Dragon and Dragon Enemy instead of using `GetComponent`.

Also, I realized the knight and knight enemy use a `gravityScale` of 5. We should probably use the same value for the dragon and dragon enemy for consistency.